File: c/array_01.c
#include<stdio.h>
void main(void)
{
    int Score[5] = {87, 66, 90, 65, 70};
    int Total_Score = 0;
    for (int i = 0; i < 5; i++) {
        printf("第%d位同学的分数: %d\n", i, Score[i]);
        Total_Score += Score[i];
    }
    printf("---------------------------------\n");
    printf("5位同学的总分数: %d\n", Total_Score);
}

File: python/array_01.py
Score = [87, 66, 90, 65, 70]
Total_Score = 0
for count in range(5):
    print('第 %d 位同学的分数: %d' %(count + 1, Score[count]))
    Total_Score += Score[count]
print('----------------------------------')
print('5位同学的总分数: %d' %Total_Score)